modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/rel/IndexScanNode.java

```diff
@@ -85,6 +89,7 @@ public IndexScanNode(
             IgniteIndex schemaIndex,
             InternalIgniteTable schemaTable,
             int[] parts,
+            long @Nullable[] terms,
```

Comment:
ups, is it good idea to mix raft internals and execution node ? probably we can some facade if it`s necessarily ?